styles/xrpl.css

```diff
@@ -9641,6 +9641,93 @@ p {
 
 .xrp-ledger-dev-portal .DocSearch-Modal {
   top: 85px;
+  background-color: #232325;
```

Comment:
This file (and the corresponding `xrpl.css.map`) shouldn't be committed to the repo; I missed that while reviewing #1581. Please delete them.